examples/red_rain.py

```diff
@@ -0,0 +1,181 @@
+# Red Rain by Matt Land, June 2016
+# ~~~~~~~~~~~
+# This is a game for the BBC Microbit, where the player moves an 'umbrella'
+# at the bottom of the LED matrix, using the A and B buttons.
+# Rain drops fall from the top of the LED matrix, and can be reflected
+# when the player successfully blocks them using the umbrella.
+#
+# Game speed will increase after the player completes each level. To complete
+# a level, block ten rain drops with the umbrella before missing five.
+# Missing five drops on one level will cause the game to end.
+#
+# Scoring is awarded based on successfully blocking rain drops.
+# Points are awarded based on current level difficulty, as well as the length
+# of the current 'streak' of successful reflections. The length of the streak does
+# not reset between levels; a long streak is the best way to maximize player score.
+#
+# Good luck!
+
+from microbit import *
+import random
+
+# sprite store for the paddles
+paddle_sprites = [
+    "66000",
+    "06600",
+    "00660",
+    "00066"
+]
+# where the paddle is on screen
+paddle_position = 0
+
+
+def run_paddle():
+    # update the paddle position based on player input
+    global paddle_position
+    if button_a.is_pressed() and paddle_position > 0:
+        paddle_position -= 1
+    if button_b.is_pressed() and paddle_position < 3:
+        paddle_position += 1
+
+# sprite store for the balls
+ball_sprites = [
+    "90000:",
+    "09000:",
+    "00900:",
+    "00090:",
+    "00009:"
+]
+
+
+class ScoreState(object):
+    # this object keeps score and increases difficulty of levels
+    _hits = 0
+    _misses = 0
+    _level = 1
+    time = 130
+    score = 0
+    _streak = 0
+
+    def hit(self):
+        # register a hit
+        self._hits += 1
+        self._streak += 1
+        self.score += self._streak * self._level * 10
+        display.show(Image("99999:90009:90009:90009:99999"))
+        sleep(25)
+
+    def miss(self):
+        # register a miss
+        self._misses += 1
+        self._streak = 0
+        display.show(Image("90009:09090:00900:09090:90009"))
+        sleep(250)
+
+    def level_complete(self):
+        if self._hits < 10:
+            return
+        image = Image("00000:00900:09990:00900:00000")
```

Comment:
display.show(Image.YES) or something else..? List of all built-in images here: 

https://microbit-micropython.readthedocs.io/en/latest/image.html#attributes